testing/integration/filter_test.go

```diff
@@ -0,0 +1,48 @@
+// +build integration
+
+package integration
+
+import (
+	"github.com/trivago/tgo/ttesting"
+	"os"
+	"strings"
+	"testing"
+)
+
+const (
+	TestConfigFileName = "test_regexp_filter.conf"
+	TmpTestFilePath    = "/tmp/gollum_test.log"
+)
+
+func TestMain(m *testing.M) {
+
+	defer teardown() // only called if we panic
+	result := m.Run()
+	teardown()
+	os.Exit(result)
+}
+
+func teardown() {
+	if _, err := os.Stat(TmpTestFilePath); err == nil {
+		os.Remove(TmpTestFilePath)
+	}
+}
+
+func TestRegexpFilter(t *testing.T) {
+	expect := ttesting.NewExpect(t)
+
+	// execute gollum
+	input := []string{"abc", "123"}
+	out, err := ExecuteGollum(TestConfigFileName, input, "-ll=2")
+
+	expect.Nil(err)
```

Comment:
expect.NoError(err) is a better choice here because the error will be written to stdout on failure.